sphinx-docs/Basic-Usage.md

```diff
@@ -295,6 +295,28 @@ In this example, the rules would permit CALDERA to only operate within the 10.24
 
 Rules can be added or modified through the GUI by navigating to *Advanced -> Sources* and clicking on '+ view rules'.
 
+### Fact Source Adjustments
+
+Fact source adjustments allow for dynamic adjustment specific ability's `visibility` in the context of an operation.
+
+**Adjustment Example (`basic` fact source)**
+
+```
+  adjustments:
+    1b4fb81c-8090-426c-93ab-0a633e7a16a7:
+      host.installed.av:
+        - value: symantec
+          offset: 3
+        - value: mcafee
+          offset: 3
+```
+
+In this example, if in the process of executing an operation, a `host.installed.av` fact was found with either the value `symantec` or `mcafee`, ability `1b4fb81c-8090-426c-93ab-0a633e7a16a7` (Sniff network traffic) would have its visibility score raised and the status `HIGH_VIZ`.  This framework allows dynamic adjustments to expected ability visibility based on captured facts (in this example the presence of anti-virus software on the target) which may impact our desire to run the ability, as it might be more easily detected in this environment.
```

Comment:
out of curiosity, will the link status be set t o`HIGH_VIZ` anytime the adjustment happens? Or only if the resulting visibility is beyond the threshold set for the operation?

Reply:
This is a great question, so far as I can tell, any anytime the adjustment happens, the status is set to `HIGH_VIZ`, regardless of the new visibility score.  My authority for this is line 396 of `planning_svc.py`, which changes the link status to `HIGH_VIZ` iff `operation.has_fact(trait=adjustment.trait, value=adjustment.value)`.

The above check sets the link's status to `HIGH_VIZ` and separately applies the visibility score adjustment.  I learn new things about core services everyday.